examples/rust/components/http-blobstore/wit/world.wit

```diff
@@ -0,0 +1,8 @@
+package wasmcloud:hello;
+
+world hello {
+  import wasi:logging/logging;
+  import wasi:blobstore/blobstore@0.1.0;
```

Comment:
This should be `wasi:blobstore/blobstore@0.2.0-draft`

Reply:
Make sure to `wit-deps update` after `blobstore` entry is added to `deps.toml`

Reply:
this is still out-of-date